cacti/README.md

```diff
@@ -75,6 +96,12 @@ The Cacti check does not include any events.
 ### Service Checks
 The Cacti check does not include any service checks.
 
+## Known issues
+
+The python library used by this integration appears to leak memory under certain circonstances. If you experience it, one workaround is to install the [`python-rrdtool`][6] package instead of `rrdtool`. This old package is not maintained anymore and is not officialy supported by this integration but some have had good results by doing that.
```

Comment:
```suggestion
The python library used by this integration appears to leak memory under certain circumstances. If you experience it, one workaround is to install the [`python-rrdtool`][6] package instead of `rrdtool`. This old package is not maintained anymore and is not officially supported by this integration but some have had good results by doing that.
```